forge/test/operators/utils/compat.py

```diff
@@ -22,6 +22,85 @@ class TestDevice:
     pass
 
 
+class TestTensorsUtils:
+
+    dev_data_format_to_dtype = {
+        # forge.DataFormat.Bfp2: torch.float16,
+        forge.DataFormat.Bfp2: torch.bool,  # mapped
+        forge.DataFormat.Bfp2_b: torch.bfloat16,
+        forge.DataFormat.Bfp4: torch.float16,
+        forge.DataFormat.Bfp4_b: torch.bfloat16,
+        forge.DataFormat.Bfp8: torch.float16,
+        forge.DataFormat.Bfp8_b: torch.bfloat16,
+
+        forge.DataFormat.Float16: torch.float16,  # mapped
+        forge.DataFormat.Float16_b: torch.bfloat16,  # mapped
+        forge.DataFormat.Float32: torch.float32,  # mapped
+
+        forge.DataFormat.Int8: torch.int8,  # mapped
+        forge.DataFormat.Lf8: torch.float16,
+        forge.DataFormat.RawUInt16: torch.uint8,
```

Comment:
Casting RawUInt16 and RawUInt32 to uint8 can lead to significant loss in precision. What is the logic behind this?

Reply:
Idea was to use unsigned number and only uint8 isn't marked as limited support.
To avoid loss of data mismatch on inputs tensors after conversion torch -> forge there is now logic of calculating intersection between torch and forge datatypes. So now tensors in torch will be created only with values that could be converted to forge without loss for integer and without significant loss for floats.

However since specifying torch types to types other that int32 cause issues, for the initial implementation I choose int32 for all integer types